fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateRoutineLoadStmt.java

```diff
@@ -110,6 +110,8 @@ public class CreateRoutineLoadStmt extends DdlStmt {
     public static final String KAFKA_PARTITIONS_PROPERTY = "kafka_partitions";
     public static final String KAFKA_OFFSETS_PROPERTY = "kafka_offsets";
     public static final String KAFKA_DEFAULT_OFFSETS = "kafka_default_offsets";
+    // optional
+    public static final String CONFLUENT_SCHEMA_REGISTRY_URL = "confluent.schema_registry.url";
```

Comment:
`kafka.schema.registry.url` 
![image](https://user-images.githubusercontent.com/44767902/212743771-6081b2b6-0d89-478e-8150-fa77c2413755.png)

Reply:
But the existing property name is in snake case.

Reply:
> `kafka.schema.registry.url` ![image](https://user-images.githubusercontent.com/44767902/212743771-6081b2b6-0d89-478e-8150-fa77c2413755.png)
We will support aws glu in the future, and confluent.schema_registry.url configuration item is only for confluent.

Reply:
@jaogoy PTAL.

Reply:
@yingtingdong PTAL.

@chaoyli I think the Schema Registry provider will take response of hive availability for it's URL. @yingtingdong  can take a check on confluent and aws.